sale_order_partial_advance/README.rst

```diff
@@ -0,0 +1,44 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
```

Comment:
PLZ take a look at https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst and adapt your file accordingly